src/layout/Input/InputComponent.tsx

```diff
@@ -24,21 +27,41 @@ export function InputComponent({
     formData?.simpleBinding ?? '',
     saveWhileTyping,
   );
-
+  const dispatch = useAppDispatch();
   const handleChange = (e) => setValue(e.target.value);
 
+  const handleChangeSearch = (e) => {
+    setValue(e.target.value);
+  };
+
   return (
-    <TextField
-      id={id}
-      onBlur={saveValue}
-      onChange={handleChange}
-      onPaste={onPaste}
-      readOnly={readOnly}
-      isValid={isValid}
-      required={required}
-      value={value}
-      aria-describedby={textResourceBindings?.description ? `description-${id}` : undefined}
-      formatting={formatting as IInputFormatting}
-    />
+    <>
+      {searchField ? (
+        <SearchField
+          id={id}
+          value={value}
+          onChange={handleChangeSearch}
+          onBlur={saveValue}
+          onPaste={onPaste}
+          aria-describedby={textResourceBindings?.description ? `description-${id}` : undefined}
+          onKeyUp={(event) => {
+            if (event.key === 'Enter') dispatch(DataListsActions.fetch());
```

Comment:
Not quite the tight coupling we want, but I guess this disappears before this PR is opened.. :-) Look into binding the `Input` to some location in the data model, and having that data model location mapped to a query parameter in the `List` component.